src/Examples/Tools/DegeneracyHunter/degeneracy_hunter.ipynb

```diff
@@ -0,0 +1,1538 @@
+    "opt.solve(m, tee=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As expected, Ipopt has no trouble solving this optimization problem."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Check if any constraint residuals are large than 1E-14"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's now inspect the new solution to see which (if any) constraints have residuals larger than 10$^{-14}$."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      " \n",
+      "All constraints with residuals larger than 1e-14 :\n",
+      "\n",
+      "count = 0 \t|residual| = 9.97185747309004e-08\n",
+      "con1 : Size=1, Index=None, Active=True\n",
+      "    Key  : Lower : Body               : Upper : Active\n",
+      "    None :  10.0 : x[0] + x[1] - x[3] :  +Inf :   True\n",
+      "variable\tlower\tvalue\tupper\n",
+      "x[0] \t\t -10 \t 1.4516741106008322 \t 10\n",
+      "x[1] \t\t -10 \t 5.061595738300216 \t 10\n",
+      "x[3] \t\t -10 \t -3.4867300513803765 \t 10\n",
+      "\n",
+      "count = 1 \t|residual| = 7.942586144338293e-09\n",
+      "con2 : Size=1, Index=None, Active=True\n",
+      "    Key  : Lower : Body             : Upper : Active\n",
+      "    None :   0.0 : x[0]*x[3] + x[1] :  +Inf :   True\n",
+      "variable\tlower\tvalue\tupper\n",
+      "x[1] \t\t -10 \t 5.061595738300216 \t 10\n",
+      "x[0] \t\t -10 \t 1.4516741106008322 \t 10\n",
+      "x[3] \t\t -10 \t -3.4867300513803765 \t 10\n",
+      "\n",
+      "count = 2 \t|residual| = 2.2426505097428162e-14\n",
+      "con3 : Size=1, Index=None, Active=True\n",
+      "    Key  : Lower : Body                         : Upper : Active\n",
+      "    None :   0.0 : x[4]*x[3] + x[0]*x[3] - x[4] :   0.0 :   True\n",
+      "variable\tlower\tvalue\tupper\n",
+      "x[4] \t\t -10 \t -1.128125759356881 \t 10\n",
+      "x[3] \t\t -10 \t -3.4867300513803765 \t 10\n",
+      "x[0] \t\t -10 \t 1.4516741106008322 \t 10\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "dict_keys([<pyomo.core.base.constraint.ScalarConstraint object at 0x14e6b6f30>, <pyomo.core.base.constraint.ScalarConstraint object at 0x14e6b6a60>, <pyomo.core.base.constraint.ScalarConstraint object at 0x14e6b6ad0>])"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "dh.check_residuals(tol=1E-14)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As expected, all of the constraints are satisfied, even with this fairly tight tolerance."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Identify all variables within 10$^{-5}$ of their bounds"
```

Comment:
As above, the Latex did not render properly in the Github review.

Reply:
I changes the (sub)section titles to not use LaTeX